backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -142,7 +144,137 @@ async def get_facility_ingestion_template_with_data(
+                    f"Found {len(fieldplan_linked_facility_ids)} facilities linked to fieldplan {fieldplan_id}")
+
+                # Fetch full facility data for fieldplan-linked facilities
+                for pf in fieldplan_facilities:
+                    facility_id = pf.get("facilityId")
+                    if facility_id:
+                        try:
+                            facility_data = facility_client.search_facility(tenant_id='in',
+                                                                            facility_id=facility_id)
+                            if facility_data and facility_data.get('facilities'):
+                                fieldplan_facilities_data.extend(facility_data.get('facilities', []))
+                        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)